contracts/README.md

```diff
@@ -0,0 +1,13 @@
+# contracts
+
+The library part of this crate contains `ethcontract` generated bindings to the smart contracts we
+use.
+
+# `vendor` Script
```

Comment:
Can you add the steps required to add a new contract (adding it in build.rs to expose it and vendor.rs to actually build it). I had some issues with it when I added the SolutionSubmission contract to dex-services (should have probably added it there myself).

Reply:
added